### PR TITLE
fix: avoid pipeline builder panic when first op is disabled

### PR DESCRIPTION
**Pull request by raj-k-singh**

### CODE REVIEW

File: pkg/query-service/app/logparsingpipeline/pipelineBuilder.go [hunk 1]
```diff
@@ -58,7 +58,7 @@ func getOperators(ops []model.PipelineOperator) []model.PipelineOperator {
 	filteredOp := []model.PipelineOperator{}
 	for i, operator := range ops {
 		if operator.Enabled {
-			if i > 0 {
+			if len(filteredOp) > 0 {
```
Comment (srikanthccv) [inline pkg/query-service/app/logparsingpipeline/pipelineBuilder.go:61]: It's hard to follow this logic. Can you help me understand what `getOperators` is supposed to do?

Comment (nityanandagohain) [reply pkg/query-service/app/logparsingpipeline/pipelineBuilder.go:61]: Yeah, it's just filtering out the disabled operators and stitching them correctly.

**Review by nityanandagohain — APPROVED** [review]

File: pkg/query-service/app/logparsingpipeline/pipelineBuilder.go (hunk 1 above)
Comment (srikanthccv) [reply pkg/query-service/app/logparsingpipeline/pipelineBuilder.go:61]: >and stitching them correctly

Can you elaborate this part?

Comment (nityanandagohain) [reply pkg/query-service/app/logparsingpipeline/pipelineBuilder.go:61]: eg:-
```
id: op1
enabled: true
output: op2

id: op2
enabled: false
output: op3

id: op3
enabled: true
```


the final config should be
```
id: op1
enabled: true
output: op3

id: op3
enabled: true
```

So here we had to updated the values of `ouput`

**Review by srikanthccv — APPROVED** [review]
